crates/rspack/tests/tree-shaking/ts-target-es5/index.ts

```diff
@@ -1,3 +1,6 @@
 export async function test() {
 
-}
\ No newline at end of file
+}
+
+
+test();
```

Comment:
I added side effects explicitly to avoid webpack shaking the entry export, cause webpack will ignore the export in the entry point